### DISCUSSION TITLE
i18next and canvas

### DISCUSSION THREAD

**Discussion by bckthomas** [i18next/i18next#1708] [Q&A]
Hi
Does anyone tried using i18next to internationalize a html5 canvas ?
I spent hours of tests but nothing works for now

Thanks

**Comment by jamuhl** [comment]
not sure what you try to do...but should be simple as `ctx.fillText(t('key'), 10, 50);`
 `

**Reply by bckthomas** [reply]
I have a language switcher in my html page and a canvas
I would like to change canvas language if I select another language.

For now, I manage to translate the first text when it loads, but it doesn't change when I change language

**Reply by jamuhl** [reply]
only, option I see is redrawing the complete canvas with the translations (using the changeLanguage callback or promise)
